bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/ConfigDescriptionRegistry.java

```diff
@@ -149,24 +174,24 @@ protected void removeConfigDescriptionProvider(ConfigDescriptionProvider configD
         List<ConfigDescriptionParameter> parameters = new ArrayList<ConfigDescriptionParameter>();
         List<ConfigDescriptionParameterGroup> parameterGroups = new ArrayList<ConfigDescriptionParameterGroup>();
 
-        boolean found = false;
-        for (ConfigDescriptionProvider configDescriptionProvider : this.configDescriptionProviders) {
-            ConfigDescription config = configDescriptionProvider.getConfigDescription(uri, locale);
-
-            if (config != null) {
-                found = true;
-
-                // Simply merge the groups and parameters
-                parameters.addAll(config.getParameters());
-                parameterGroups.addAll(config.getParameterGroups());
+        boolean found = fillFromProviders(uri, locale, parameters, parameterGroups);
+        URI alias = null;
+        if (!found) {
+            for (ConfigDescriptionAliasProvider aliasProvider : configDescriptionAliasProviders) {
+                alias = aliasProvider.getAlias(uri);
+                if (alias != null) {
+                    logger.debug("No config description found for '{}', using alias '{}' instead", uri, alias);
+                    found = fillFromProviders(alias, locale, parameters, parameterGroups);
+                    break;
```

Comment:
Shouldn't this `break` only be called if `found == true`?

Reply:
This was intended initially, but on a second thought I can't explain why it _should_ be like that. I certainly doesn't hurt to continue until the first "successful" alias, if there are multiple.